fatima-krystle_talens-getEven.js

```diff
@@ -0,0 +1,81 @@
+//Get Even
+
+for (var i=0; i<=200; i+=2){
+    console.log(i)
+}
+
+
+//Excited Kitten
+var catSays = ["...human...why you taking pictures of me?...", "...the catnip made me do it...", "meow?", "...why does the red dot always get away..."]
+
+for (var i=1; i<=10; i++){
```

Comment:
nice thought to start the loop at 10 to make your conditional in the next line simpler with regards to the first cat phrase 💯